ocaml/idl/datamodel_pool.ml

```diff
@@ -803,6 +803,14 @@ open Datamodel_types
       ~allowed_roles:_R_POOL_OP
       ()
 
+  let deconfigure_repository_proxy = call
```

Comment:
I think "deconfigure" is a quite unusual verb. I would have preferred any of "decommission", "disable", "remove". Another option is "clear", which has a different meaning.

Reply:
Thanks. I prefer to use "remove" in this case.

Reply:
But the SDK has constraint on the usage of "remove". So I change to use "reset".

Reply:
The updated code uses "reset", though. So maybe we should get an idea what this code is supposed to do first. What happens when this API call is executed and how does it affect future API calls?

Reply:
The call is just to remove the configured proxy, or in other words, change it to the initial default value which stands for "no proxy" configured.

Reply:
I think "remove" would be most appropriate, followed by "disable". "Reset" and "clear" would suggest the cache still exists, is initially empty, but would fill up again. If the cache does not fill up again, it is disabled.

Reply:
It's not actually a cache, but a network proxy. "Reset" to me means that it's not necessarily going away. Perhaps "disable" is the best if "remove" isn't possible.